src/TensorFlow.jl

```diff
@@ -165,14 +165,13 @@ library loaded.
 macro py_proc(expr)
     quote
         eval(Main, quote
-            remotecall_fetch($(TensorFlow.load_python_process())) do
+            remotecall_wait($(TensorFlow.load_python_process())) do
```

Comment:
This can be just `remotecall` I think.
Because eventually the consumer is going to`fetch` it.
And `fetch` always does a `wait` until there is a result